client_sdk/go/fpc/contract.go

```diff
@@ -30,75 +29,76 @@ type ContractInterface interface {
 func GetContract(network *gateway.Network, chaincodeId string) ContractInterface {
 	contract := network.GetContract(chaincodeId)
 	ercc := network.GetContract("ercc")
-	return &Contract{contract, ercc, nil, nil}
+	return &Contract{
+		contract:     contract,
+		ercc:         ercc,
+		enclavePeers: nil,
+		ep: &crypto.EncryptionProviderImpl{GetCcEncryptionKey: func() ([]byte, error) {
+			return ercc.EvaluateTransaction("queryChaincodeEncryptionKey", chaincodeId)
+		}}}
 }
 
 type Contract struct {
-	contract                     *gateway.Contract
-	ercc                         *gateway.Contract
-	cachedChaincodeEncryptionKey []byte
-	enclavePeers                 []string
+	contract     *gateway.Contract
+	ercc         *gateway.Contract
+	enclavePeers []string
```

Comment:
This is presumably a list of endpoints, i.e., `host:port`.  Maybe then better call it also `endpoints` or `peerEndPoints` (or otherwise add a comment on format/semantics?)